logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -884,6 +885,12 @@ internal class ConversationDataSource internal constructor(
             }
         }
 
+    override suspend fun deleteLocalConversation(conversationId: ConversationId): Either<CoreFailure, Unit> {
```

Comment:
I was thinking between `deleteLocalConversation` and `deleteConversationLocally` - which one would you guys prefer?

Reply:
Also there is the `deleteConversation` but it invoked mls method to nuke the conversation which I think we dont want while removing it locally? Am I right?

Reply:
I would prefer `deleteConversationLocally`, we have other similar methods implemented already like `deleteAssetLocally` or `updateProtocolLocally` so it would make more sense 🤔 
About using `deleteConversation`, I think that would try to delete it globally for everyone, if so then we shouldn't use that in this case 😄

Reply:
Gotta rename it to `locally` then 👍